app/features/atp/passport/ui/idx/controller.ts

```diff
@@ -193,12 +193,21 @@ const postPassport = async (
     identityEvidence.atpResponse = decoded;
     req.session.identityEvidence = req.session.identityEvidence || [];
     req.session.identityEvidence.push(identityEvidence);
-
     addToFill(req, "dob", {
```

Comment:
Looking at these existing lines of code on running my machine, I'm not seeing this "fill from history" feature working on the date fields at all. I don't know if that's a bug, or because they're a weird 3-fields-merged-together sort of thing it can't work.

🤷 

If it can't work on these sorts of fields, we should remove the feature from these fields; if it can work, we should fix it and use your improvements. 

Let's investigate this together.